app/forms/collection_form.rb

```diff
@@ -14,35 +18,20 @@ def validate(params)
 
   # Copies the values to the model and saves and indexes
   def save
-    if model.new_record?
-      @model = register_model
-    else
-      # update case
-      # TODO: after cocina updater supports descriptive metadata, make a cocina update call instead
-      sync
-    end
-    model.save
-    # update index immediately. Do not pass Go. Do not collect $200.
-    Argo::Indexer.reindex_pid_remotely(model.pid)
+    @model = register_model
   end
 
-  # Copies the abstract value to the model descMetadata
-  def sync
-    # NOTE: collection_abstract only appears in conjunction with collection_title
-    #       and disjoint from collection_catkey
-    return if params[:collection_abstract].blank?
+  attr_reader :errors, :model, :params
 
-    model.descMetadata.abstract = params[:collection_abstract]
-  end
+  delegate :model_name, :to_key, :to_model, :new_record?, to: :model
 
   private
 
-  # @return [Dor::Collection] registers the Collection
+  # @return [Cocina::Models::Collection] registers the Collection
   def register_model
-    response = Dor::Services::Client.objects.register(params: cocina_model)
-    WorkflowClientFactory.build.create_workflow_by_name(response.externalIdentifier, 'accessionWF', version: '1')
-    # Once it's been created we populate it with its metadata
-    Dor.find(response.externalIdentifier)
+    Dor::Services::Client.objects.register(params: cocina_model).tap do |response|
+      WorkflowClientFactory.build.create_workflow_by_name(response.externalIdentifier, 'accessionWF', version: '1')
```

Comment:
I get that this is the code we already had here, but also I wonder about creating accessionWF this way vs. using `Dor::Services::Client.object(response.externalIdentifier).accession.start(workflow: 'accessionWF')`. When should we reach for one over the other?

Reply:
I don't know. I didn't look into that. Would you like to create an issue?

Reply:
I don't know if it's issue-worthy. There's a ton of `WorkflowClientFactory` in Argo, so I'm not sure there's much value in replacing this one call with a DSC call.